tr.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE HTML>
+<!--
+	Dimension by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html lang="en">
```

Comment:
```suggestion
<html lang="tr">
```

If this is the right iso code add this one otherwise change it